10.11/Dockerfile.rhel7

```diff
@@ -0,0 +1,80 @@
+FROM rhscl/s2i-core-rhel7
+
+# MariaDB image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MariaDB
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+
+ENV MYSQL_VERSION=10.11 \
+    APP_DATA=/opt/app-root/src \
+    HOME=/var/lib/mysql \
+    SUMMARY="MariaDB 10.11 SQL database server" \
+    DESCRIPTION="MariaDB is a multi-user, multi-threaded SQL database server. The container \
+image provides a containerized packaging of the MariaDB mysqld daemon and client application. \
+The mysqld server daemon accepts connections from clients and provides access to content from \
+MariaDB databases on behalf of the clients."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="MariaDB 10.11" \
+      io.openshift.expose-services="3306:mysql" \
+      io.openshift.tags="database,mysql,mariadb,mariadb1011,rh-mariadb1011" \
+      com.redhat.component="rh-mariadb1011-container" \
+      name="rhscl/mariadb-1011-rhel7" \
+      version="1" \
+      com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#rhel" \
+      usage="docker run -d -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 rhscl/mariadb-1011-rhel7" \
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+
+EXPOSE 3306
+
+# This image must forever use UID 27 for mysql user so our volumes are
+# safe in the future. This should *never* change, the last test is there
+# to make sure of that.
+RUN yum install -y yum-utils && \
+    prepare-yum-repositories rhel-server-rhscl-7-rpms && \
+    INSTALL_PKGS="rsync tar gettext hostname bind-utils groff-base rh-mariadb1011 rh-mariadb1011-mariadb-server-utils rh-mariadb1011-mariadb-syspaths" && \
```

Comment:
I am not mariadb and container expert so maybe I am mistaken, but AFAIK rhscl does not support mariadb10.11. At lease I haven't found such branch in dist-git

Reply:
Fixed https://github.com/sclorg/mariadb-container/pull/223/commits/746dc057b928981308fd521d70ddf3ae7ece29f1. But there is an exclude-rhel7 file that means, that the version is not build neither tested.

Reply:
Why are we even adding the Dockerfile if there is no need for it?

Reply:
Fixed by https://github.com/sclorg/mariadb-container/pull/223/commits/8603271a2f27714811a83a9d68f832210c85d880